.vscode/settings.json

```diff
@@ -1,11 +1,11 @@
 {
-    "python.pythonPath": ".tox\\dev\\Scripts\\python.exe",
+    "python.defaultInterpreterPath": ".tox/dev/bin/python",
```

Comment:
is this intended?

Reply:
yes it is.
`python.pythonPath` is not a valid setting for VS Code anymore. It was renamed to `python.defaultInterpreterPath`.
And I guess they can now "translate" the different paths from UNIX to Windows.